packages/core/src/plugin-state/child.tsx

```diff
@@ -17,10 +18,18 @@ function PluginPropsDocument<Props extends Record<string, unknown>>({
   props?: Props
   parentProps: unknown
 }) {
+  const { scope, editable } = React.useContext(ScopeContext)
   const pluginProps = React.useMemo(() => {
     return { ...props, parent: parentProps }
   }, [props, parentProps])
-  return <Document pluginProps={pluginProps} id={id} />
+  return (
+    <Document
+      pluginProps={pluginProps}
+      id={id}
+      scope={scope}
```

Comment:
Review usage of scope & editable. Might be easier to use the contexts in `Document` instead, not sure

Reply:
Yep, that was easier (especially with the merged context containing both editable and scope), see https://github.com/edtr-io/edtr-io/pull/136/commits/cae9778852efa3f3b88ee1bd4b41a5643ddfb4b9